CHANGELOG.md

```diff
@@ -10,6 +10,21 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+## [0.3.1] - 2015-07-09
+### Fixed
+- Fixed directory permission for custom buildpacks
+
+### Added
+- Added ability to build in docker container
+
+### Removed
+- Remove testing play-v1 from play buildpack
```

Comment:
We just removed the test not the support. The scala buildpack handles play-v1 as I understand it anyway.